char input
---
float64 acceleration
